heartbeat/mysql-common.sh

```diff
@@ -94,8 +96,12 @@ MYSQL_BINDIR=`dirname ${OCF_RESKEY_binary}`
 # Convenience variables
 
 MYSQL=$OCF_RESKEY_client_binary
+if [ "$OCF_RESKEY_replication_require_ssl" = "true" ]
+then
+  MYSQL_OPTIONS_LOCAL_SSL_OPTIONS="--ssl"
```

Comment:
Could you set the else branch to "" as well? this way bash won't complain if we run the agent with bash -u

Reply:
👍 Done.